spec/list_chores_spec.js

```diff
@@ -0,0 +1,25 @@
+// This spec will test whether we successfully return a list of chores
```

Comment:
I don't think this is true anymore. Should we have comments like this in our code? I think our code should describe itself (unless it's particularly complex for some reason).